runironic.sh

```diff
@@ -11,7 +11,9 @@ rm -rf /shared/log/ironic
 
 mkdir -p /shared/log/ironic
 
-/usr/bin/ironic-conductor ${IRONIC_CONFIG_OPTIONS} &
-/usr/bin/ironic-api --config-file /usr/share/ironic/ironic-dist.conf ${IRONIC_CONFIG_OPTIONS} &
+python3 -c 'import os; import sys; import jinja2; sys.stdout.write(jinja2.Template(sys.stdin.read()).render(env=os.environ))' < /etc/httpd-ironic-api.conf.j2 > /etc/httpd/conf.d/ironic.conf
+sed -i "/Listen 80/c\#Listen 80" /etc/httpd/conf/httpd.conf
+exec /usr/sbin/httpd 
```

Comment:
All commands after exec won't be executed. It may be the right time to remove this file for good.